<!doctype html>  
<html lang="en">  
<head>  
  <meta charset="utf-8">  
  <title>Procedure for the production of exomic DNA libraries for Illumina sequencing</title>  
  <meta name="Daniel Brown" content="Cancer Signaling">  
  <link rel="stylesheet" type="text/css" href="plain.css" />
  <!--[if lt IE 9]>  
  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>  
  <![endif]-->
</head>  
<body>
<h1>Procedure for the production of exomic DNA libraries for Illumina sequencing<h1>
<h2>You will need the following consumables</h2>
<ul>
  <li>1ug of high quality genomic DNA per sample</li>
  <li>NEBnext DNA library preparation kit for Illumina. Only need up to adaptor ligation</li>
  	<ol>
  		<li>Illumina adaptors 1 - 12 (kit A)</li>
  	</ol>
  <li>Qiaquick PCR clean up columns or equivalent</li>
  <li><b>550uL</b> <i>Recommended, may change for size selection</i> AMpure XP beads. Bring to room temperature before use</li> <!--Check the amount of Ampure beads I need here-->
  <li><b>Fresh</b> 80% ethanol for use with the beads</li>
  <li>SeqCap EZ reagent kit. This contains human COT DNA, primers, PCR master mix and hybridisation/ wash buffers.</li>
  <li>Dynabeads M-270 strepavidin beads for library capture</li>
  <li>qPCR reagents for enrichment quantification</li>
  	<ol>
  		<li>At least 4 primer pairs for NSC control regions. Consider ordering custom regions.</li>
  	</ol>
  <li>PCR grade water</li>
</ul>

<h2>Exome capture sample sheet</h2>
		 <table border="1" cellpadding="15">
			<tr>
				<th>Index</th>
				<th>Sample</th>
				<th>Index</th>
				<th>Sample</th>
				<th>Index</th>
				<th>Sample</th>
			</tr>
			<tr>
				<td>1</td>
				<td>.................................................................</td>
				<td>5</td>
				<td>.................................................................</td>
				<td>9</td>
				<td>.................................................................</td>
			</tr>
			<tr>
				<td>2</td>
				<td>.................................................................</td>
				<td>6</td>
				<td>.................................................................</td>
				<td>10</td>
				<td>.................................................................</td>
			</tr>
			<tr>
				<td>3</td>
				<td>.................................................................</td>
				<td>7</td>
				<td>.................................................................</td>
				<td>11</td>
				<td>.................................................................</td>
			</tr>
			</tr>
			<tr>
				<td>4</td>
				<td>.................................................................</td>
				<td>8</td>
				<td>.................................................................</td>
				<td>12</td>
				<td>.................................................................</td>
			</tr>
			</table>
</br>
  <h2>Fragmentation of gDNA. <i>(Gel-free method)</i>...................................................................................................<input type='checkbox' align='right'></h2>
  <p>Have Illumina resuspension buffer thawed and on ice before shearing.</br>
  	Illumina recommends to normalise the gDNA samples to a final volume of 55 μl at 20 ng/μl into each 
  	well of the new 0.3 ml tube labeled with the DNA barcode.</p>
  <ol>
  	<li><b>Use a Covaris shearer with the following settings:</b></li>
  		<ul>
  			<li>Time: 120 seconds </li>
  			<li>Temperature: 6 degrees</li>
  			<li>Duty factor:</li>
  			<li>Peak incident power:</li>
  			<li>Cycles per burst:</li>
  			<li>Duration:</li>
  		</ul>
  	<li>Seal the Covaris tubes and centrifuge to 600 xg for 1 minute. Transfer 50 μl of fragmented DNA from each Covaris tube</li>
  	</ol>
  	
  	
 <h6 align='right'>Written by Dan Brown 08 May 2013<h6>

</body>  
</html>  
