<!doctype html>  
<html lang="en">  
<head>  
  <meta charset="utf-8">  
  <title>Procedure for the production of exomic DNA libraries for Illumina sequencing</title>  
  <meta name="Daniel Brown" content="Cancer Signaling">  
  <link rel="stylesheet" type="text/css" href="plain.css" />
  <!--[if lt IE 9]>  
  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>  
  <![endif]-->
</head>  
<body>
<h1>Procedure for the production of exomic DNA libraries for Illumina sequencing<h1>
<h2>You will need the following consumables</h2>
<ul>
  <li>1ug of high quality genomic DNA per sample</li>
  <li>NEBnext DNA library preparation kit for Illumina. Only need up to adaptor ligation</li>
  	<ol>
  		<li>Illumina adaptors 1 - 12 (kit A)</li>
  	</ol>
  <li>Qiaquick PCR clean up columns or equivalent</li>
  <li><b>550uL</b> <i>Recommended, may change for size selection</i> AMpure XP beads. Bring to room temperature before use</li> <!--Check the amount of Ampure beads I need here-->
  <li><b>Fresh</b> 80% ethanol for use with the beads</li>
  <li>SeqCap EZ reagent kit. This contains human COT DNA, primers, PCR master mix and hybridisation/ wash buffers.</li>
  <li>Dynabeads M-270 strepavidin beads for library capture</li>
  <li>qPCR reagents for enrichment quantification</li>
  	<ol>
  		<li>At least 4 primer pairs for NSC control regions. Consider ordering custom regions.</li>
  	</ol>
  <li>PCR grade water</li>
</ul>

<h2>Exome capture sample sheet</h2>
		 <table border="1" cellpadding="15">
			<tr>
				<th>Index</th>
				<th>Sample</th>
				<th>Index</th>
				<th>Sample</th>
				<th>Index</th>
				<th>Sample</th>
			</tr>
			<tr>
				<td>1</td>
				<td>.................................................................</td>
				<td>5</td>
				<td>.................................................................</td>
				<td>9</td>
				<td>.................................................................</td>
			</tr>
			<tr>
				<td>2</td>
				<td>.................................................................</td>
				<td>6</td>
				<td>.................................................................</td>
				<td>10</td>
				<td>.................................................................</td>
			</tr>
			<tr>
				<td>3</td>
				<td>.................................................................</td>
				<td>7</td>
				<td>.................................................................</td>
				<td>11</td>
				<td>.................................................................</td>
			</tr>
			</tr>
			<tr>
				<td>4</td>
				<td>.................................................................</td>
				<td>8</td>
				<td>.................................................................</td>
				<td>12</td>
				<td>.................................................................</td>
			</tr>
			</table>
</br>

<h2>End repair sheared DNA..................................................................................................................................<input type='checkbox'></h2>
  <p>Starting material 1ug of sheared gDNA.</p>
  <ol>
    <li>Prepare the following reagents <i style="color:green;">Green lids</i>:</br>
    <table border="1" cellpadding="10">
			<tr>
				<th>Component</th>
				<th>Volume (uL)</th>
			</tr>
			<tr>
				<td>End prep Enzyme Mix</td>
				<td>3.0</td>
			</tr>
			<tr>
				<td>End Repair Reaction Buffer (10X)</td>
				<td>6.5uL</td>
			</tr>
			<tr>
				<td>Fragmented DNA</td>
				<td>55.5uL</td>
			</tr>
			<tr>
				<td>Total volume</td>
				<td>65.0</td>
			</tr>
	</table></br>	
	<li>Pre-heat the thermal cycler to 30°C. Pre-heat the lid to 100°C.</li>
	<li>Perform the following incubations in a PCR machine:</li>
	<ul>
		<li>20°C for 30 minutes</li>
		<li>65°C for 30 minutes</li>
		<li>4°C hold</li>
	</ul>
	<li>Proceed immediately to Ligate Adapters. <i>NEBnext ultra has no post-reaction clean-up</i>.</li>
  </ol>
		
<h2>Ligate adaptors................................................................................................................................................<input type='checkbox'></h2>
	<p>Cleanup and size select using AmpureXP beads. Have these at room temp for 30 min prior to use</p>
	<ol>
		<li>Prepare the following reagents <i style="color:red;">Red lids</i></br>
    <table border="1" cellpadding="10">
			<tr>
				<th>Component</th>
				<th>Volume (uL)</th>
			</tr>
			<tr>
				<td>End Prep reaction mixture</td>
				<td>65.0</td>
			</tr>
			<tr>
				<td>Blunt/TA Ligase Master Mix</td>
				<td>15.0</td>
			</tr>
			<tr>
				<td>NEBNext Adapter for Illumina</td>
				<td>2.5uL</td>
			</tr>
			<tr>
				<td>Ligation Enhancer</td>
				<td>1.0uL</td>
			</tr>
			<tr>
				<td>Total volume</td>
				<td>83.5</td>
			</tr>
	</table></br>
		<li>Incubate at 20°C for <i>15 minutes</i> in a thermal cycler.</li>
		<li>Add 3μl of USER enzyme to each reaction. Mix thoroughly.</li>
		<li>Incubate at 37°C for <i>15 minutes</i> in a thermal cycler.</li>
	</ol>
	
<h2>Size selection...............................................................................................................................................<input type='checkbox'></h2>
	<p>The following size selection protocol is for libraries with 200 bp inserts only.</p>
	<ol>
			<li>To the ligation reaction (83.5 μl), add 16.5 μl nuclease-free water to bring the reaction volume to 100 μl.</li>
			<li>Add 60 μl (0.6X) resuspended AMPure XP beads and mix well on a vortex mixer or by pipetting up and down at least 10 times.</li>
			<li>Incubate at room temperature for 5 minutes.</li>
			<li>Place the tube on an appropriate magnetic rack to separate beads from the supernatant. 
			After the solution is clear (about 5 minutes), carefully transfer the supernatant containing your DNA 
			to a new tube <b>(Caution: do not discard the supernatant)</b>.</li>
			<li>Add 25 μl (0.25X) resuspended AMPure XP beads to the supernatant (160 μl), mix well and incubate for 5 minutes at room temperature.</li>
			<li>Wash 2 times with 200uL 80% ethanol.</li>
			<li>Air dry beads for 10 minutes while the tube is on the magnetic rack with the lid open.</li>
			<li>Elute DNA target from the beads with 25 μl nuclease-free water. Mix well on a vortex mixer or by pipetting up and down, then put the tube in the magnetic rack until the solution is clear.</li>
			<li>Without disturbing the bead pellet, transfer 20 μl of the supernatant to a clean PCR tube, and proceed to PCR enrichment. Contamination of beads will inhibit downstream PCR.</li>
		</ol>
	
<h2>QC the pre-amplified libraries...........................................................................................................................<input type='checkbox'></h2>
    <p>This step is optional to make sure the library prep worked. In order of workflow and information extracted.</p>
    <ul>
    	<li>Qubit</li>
    	<li>Bioanalyser DNA chip</li>
    	<li>qPCR of NS loci</li>
  	
<p>Proceed to the first step of library preparation <a href="130611_nebPCRqcCheck.html">The End repair and Adapter Ligation protocol</a> </p>
  	
 <h6 align='right'>Written by Dan Brown 08 May 2013<h6>

</body>  
</html>  
