<!doctype html>  
<html lang="en">  
<head>  
  <meta charset="utf-8">  
  <title>Procedure for production of cDNA libraries for RNA-seq</title>  
  <meta name="Daniel Brown" content="Cancer Signaling">  
  <link rel="stylesheet" type="text/css" href="plain.css"/>
  <!--[if lt IE 9]>  
  <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>  
  <![endif]-->
</head>  
<body>
<script type="text/javascript" src="130625_menuBar.js"> </script>
<h1>Procedure for production of cDNA libraries for RNA-seq</h1>
<p>This protocol will produce strand specific cDNA libraries with low RNA input for sequencing on the Illumina platform.</p>
<h2>You will need (for 6 samples)</h2>
<ul>
  <li>50 - 250ng of high quality RNA </li>
  <li>Actinomycin D. Stock 5ug/uL to 0.1ug/uL.</li>
  <li>2x DNA clean up column per sample (eg QIAquick)</li>
  <li>85uL (550uL) AMpure XP beads. Bring to room temperature before use</li>
  <li>6mL fresh 80% ethanol for use with the beads</li>
  <li>Unique index primers if you intend on multiplexing. </br>The NEB multiplex oligos for Illumina are the same adaptor sequence as the in-house Illumina ones</li>
 </ul> 
  
<table border="1" align="center" cellpadding="15">
			<tr>
				<th>Index</th>
				<th>Sample</th>
				<th>Index</th>
				<th>Sample</th>
			</tr>
			<tr>
				<td>1</td>
				<td>.................................................................</td>
				<td>4</td>
				<td>.................................................................</td>
			</tr>
			<tr>
				<td>2</td>
				<td>.................................................................</td>
				<td>5</td>
				<td>.................................................................</td>
			</tr>
			<tr>
				<td>3</td>
				<td>.................................................................</td>
				<td>6</td>
				<td>.................................................................</td>
			</tr>
			</tr>
			</table>
  
<h3>First strand cDNA synthesis...............................................................................................<input type='checkbox'></h3>
  <p>Take the prepared mixture from the <a href="130606_RNAextraction.html">mRNA purification and shearing</a>
  Incubate the samples as follows:
  <ol>
  	<li>10 min at 25 °C</li>
  	<li>15 min at 42 °C</li>
  	<li>15 min at 70 °C</li>
  	<li>Hold at 4 °C</li>
  </ol>
  It is not necessary to clean up the reaction afterwards.</p>
  
<h3>2nd strand synthesis.................................................................................................................<input type='checkbox'></h3>
 <ol>
 	<li>Add the following to the first strand synthesis reaction.</li>
 	 <table border="1" align="center" cellpadding="15">
			<tr>
				<th>Reagent</th>
				<th>x1 volume (uL)</th>
				<th>x6 volume + 10% (uL)</th>
			</tr>
			<tr>
				<td style="color:#FF6600;">2nd strand synthesis mix with dUTP</td>
				<td >8</td>
				<td>52.8</td>
			</tr>
			<tr>
				<td style="color:#FF6600;">Second strand synthesis enzyme mix</td>
				<td>4</td>
				<td>24.6</td>
			</tr>
			<tr>
				<td>Nuclease free water</td>
				<td>48</td>
				<td>316.4</td>
			</tr>
			</table>
	<li>Mix by pipette.</li>
	<li>Incubate in a thermocycler for 1 hour at 16 C. <i>The samples can be safely stored at 4 C overnight as cDNA.</i></li>
	<li>Clean up the reaction using the Machery Nagel Gel - PCR clean up kit. Elute in 60uL water.</li>		
 </ol>
 
<h3>End repair library..........................................................................................................................................<input type='checkbox'></h3>
<p>Use a new PCR tube and incubate in a thermocycler:
  <table border="1" align="center" cellpadding="15">
			<tr>
				<th>Reagent</th>
				<th>x1 volume (uL)</th>
				<th>x6 volume + 10% (uL)</th>
			</tr>
			<tr>
				<td style="color:#339900;">End repair reaction buffer</td>
				<td >6.5</td>
				<td>42.9</td>
			</tr>
			<tr>
				<td style="color:#339900;">End prep repair enzyme mix</td>
				<td>3</td>
				<td>19.8</td>
			</tr>
			<tr>
				<td>cDNA</td>
				<td>55.5</td>
				<td>NA</td>
			</tr>
			</table>
<ol>
	<li>30 minutes at 20 °C</li>
	<li>30 minutes at 65 °C</li>
	<li>Hold at 4 °C</li>
</ol>
Proceed <b>directly</b> to adapter ligation.</li>
</p>

<h3>Adapter ligation..............................................................................................................................<input type='checkbox'></h3>
<p>Use 65uL of the reaction from A tailing and perform in a PCR machine set to 20 C and paused.</br>
  <table border="1" align="center" cellpadding="15">
			<tr>
				<th>Reagent</th>
				<th>x1 volume (uL)</th>
				<th>x6 volume + 10% (uL)</th>
			</tr>
			<tr>
				<td style="color:#990000;">Blunt/TA ligase Mix</td>
				<td >15</td>
				<td>99</td>
			</tr>
			<tr>
				<td style="color:#990000;">NEBnext adapter</td>
				<td>1</td>
				<td>6.6</td>
			</tr>
			<tr>
				<td>Water</td>
				<td>2.5</td>
				<td>16.5</td>
			</tr>
			</table>
<ol>
	<li>15 minutes at 20 °C</li>
	<li>Hold at 4 °C</li>
</ol>
The adapter comes from the NEBnext multiplex oligo kit. The USER excision step is performed during the reaction.
</p>

<h3>Size selection........................................................................................................................................<input type='checkbox'></h3>
<p>Another critical step.</b> Make sure the <b>AmpureXP beads are at room temp</b> and well mixed. Make up 80% ethanol <b>fresh</b></br>
It is best to conduct size selection in 0.2mL PCR tubes and a ring magnet.</p>
<ol>
	<li>To 83.5uL of the ligation reaction add 16.5uL of nuclease free water to bring the reaction volume to 100uL.</li>
	<li>Add 60uL <b>(0.6x)</b> resuspended AMPure XP beads and mix by a vortex. Incubate for 5 minutes at room temperature.</li>
	<li>Spin the tube on a tommy and place on a magnetic rack for 5 minutes or until the solution is clear.
	</br><b>Transfer the supernatant</b> to a clean tube and keep it! Retain the beads only for insurance purposes.</li>
	<li>Add 25uL <b>(0.25x)</b> resuspended AMPure XP beads to 160uL of the supernatant. Mix well and incubate for 5 minutes at room temperature.</li>
	<li>Spin the tube on a tommy and place on a magnetic rack to separate beads from the supernatant. After 5 minutes or when the solution is 
	clear <b>carefully remove</b> and discard the <b>supernatant</b>. Do not disturb the beads.</li>
	<li>Add 200uL of freshly prepared ethanol to the tube while in the magnetic rack. Incubate for 30 sec and then discard the supernatant.</li>
	<li>Do another wash</li>
	<li>Briefly spin the tube in a tommy and put back into the magnetic rack.</li>
	<li>Completely remove the ethanol and air dry beads with the lid open for 10 minutes at room temperature. It is critical not to overdry the beads as they will crack.</li>
	<li>Elute the target cDNA from the beads with 25uL of nuclease-free water. Mix well and spin in a tommy.</li>
	<li>Without disturbing the beads transfer 20uL of the supernatant to a clean PCR tube. It is <b>critical</b> not to carry any beads over to the PCR enrichment as it will fail. If necessary perform a second magnetic incubation.</li>
</ol>

 <!--Up to here-->

<h3>USER excision, indexing and PCR enrichment.....................................................................................<input type='checkbox'></h3>
<p>Make sure the right tube has the right index added to it!</br>
 The cycle number can be between 12 - 15 cycles depending on the input mRNA. 15 cycles is recommended for 10ng mRNA.</p>
 <table border="1" align="center" cellpadding="15">
			<tr>
				<th>Reagent</th>
				<th>x1 volume (uL)</th>
				<th>x6 volume + 10% (uL)</th>
			</tr>
			<tr>
				<td style="color:#CC0000;">USER enzyme</td>
				<td >3</td>
				<td>99</td>
			</tr>
			<tr>
				<td style="color:#000099;">High fidelity PCR master mix 2x</td>
				<td>25</td>
				<td>6.6</td>
			</tr>
			<tr>
				<td style="color:#000099;">Universal PCR primer 25uM</td>
				<td>1</td>
				<td>16.5</td>
			</tr>
			<tr>
				<td style="color:#000099;">Index primer 25uM <b>unique!</b></td>
				<td>1</td>
				<td>16.5</td>
			</tr>
			<tr>
				<td>Size selected cDNA</td>
				<td>20</td>
				<td>16.5</td>
			</tr>
			</table>
			
<p>PCR cycling conditions.</p>
<ol>
	<li>User digest 15 min @ °37 C</li>
	<li>Denature 10 sec @ 98 °C</li>
	<li>Cycling
	<ul>
		<li>Denature 10 sec @ 98 °C</li>
		<li>Anneal 30 sec @ 65 °C</li>
		<li>Extend 30 sec @ 72 °C</li>
	</ul></li>
	<li>Final extension 5 min @ 72 °C</li>
	<li>Hold 4 °C</li>
</ol>			
<p>Purify the PCR product by AMPure XP beads. Putting the product through a column doesn't remove adapter dimer <b>Elute in 20uL water</b>.</p>			
 
 <h3>Library QC before sequencing</h3>
<p>Measure a 1:4 dilution of the library by Qubit. Then dilute again to 1,000 pM for a bioanalyser run.</br>
NEB recommends diluting each library 1:4 and assessing the size distribution of the library on a high sensitivity chip bioanalyser.</br> 
Seqanswers suggests bioanalyser alone is not very quantitative</p>
 
 <h4>Appendix: Some tips for when handling beads</h4>
<ul>
<li>When opening tube lids on the magnet, hold down the tube so liquid doesn't splash everywhere</li>
<li>If the pellet is diffuse, especially during enzymatic clean-up give the tube a jiggle when still in the magnet</li>
<li>Suck up supernatant slowly as to not disturb the beads</li>
<li>For the final elution it is critical not to contaminate the PCR with beads. It is better to elute twice</li>
<li>The volume of beads for size selection is critical. Measure the volume of DNA by pipette. It is more accurate to dial down.</li>
<li>Top up the solution to 150uL if required</li>
</ul>

 <h6 align='right'>Written by Dan V Brown 6 June 2013<h6>

</body>  
</html>  
